index.js

```diff
@@ -189,7 +189,30 @@ class DHT extends EventEmitter {
     })
   }
 
-  _addNode (id, peer, token) {
+  get holepunchable () {
```

Comment:
does it have to be a getter?

Reply:
Good question! My initial reason was the it is pure logic, ie could be calculated when a peer is updated easily also, so that sounded like a getter to me.

You think it's easier to understand with a function?